bootstrap_venv.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ex
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $DIR
+python3 -m venv venv
+. venv/bin/activate
+pip install -U pip
+pip install -U wheel
+pip install -U cmake ninja
+
+FARM_NG_CMAKE_PATH="${1:-.}"
+
+mkdir -p build.venv
+cd build.venv
+cmake \
+    -DFARM_NG_DEV_BUILD=On \
+    -Dfarm_ng_INSTALL_PREFIX=$DIR/venv \
```

Comment:
Likely we should install this somewhere else other than the venv, so that its a relocated package of only what's built by cmake.